parastell/cubit_io.py

```diff
@@ -205,3 +205,14 @@ def export_dagmc_cubit_native(
     # exports
     if delete_upon_export:
         cubit.cmd(f"delete mesh volume all propagate")
+
+
+def create_new_cubit_instance():
```

Comment:
I thin it would be helpful to inser this immediately after `init_cubit()` so that folks can find it and associate it with that method more easily.

Reply:
Yeah this makes sense; I'll insert it after init_cubit().